modules/command/dh/dh-read-data-location-request-command.js

```diff
@@ -58,8 +58,28 @@ class DHReadDataLocationRequestCommand extends Command {
             return Command.empty();
         }
 
+        const validImports = [];
+        for (let i = 0; i < imports.length; i += 1) {
+            if (imports[i].data_provider_wallet.toLowerCase()
+                === this.config.node_wallet.toLowerCase()) {
+                // eslint-disable-next-line no-await-in-loop
+                const offer = await Models.offers.findOne({
+                    attributes: ['offer_id'],
+                    where: {
+                        data_set_id: {
+                            [Op.eq]: imports[i].data_set_id,
```

Comment:
You don't have to use Op.eq it's enough just this:
where: {
data_set_id: imports[i].data_set_id
}